CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contribution Guidelines
+
+Thanks for taking the time to contribute. You're the best! Around! Nothing's gonna ever keep you down.
+
+✨ 🎉 ✨
+
+## Code of conduct
+
+This project is governed by the [Contributor Covenant Code of Conduct](CODE_OF_CONDUCT.md). We want to cultivate a respectful and constructive environment for everyone involved, and we expect all other contributors to do the same. Please use Github's reporting features if you have any concerns or reach out to us directly via our [Discord server](https://discord.gg/gMfXeXR).
+
+## Asking general questions
+
+Discord is generally the fastest way to get in touch with us if you have general questions. We try to answer questions in a timely manner. If you have a bug report or feature request, feel free to discuss it with us there or just open an issue with as much detail as possible.
+
+## Ways to contribute
+
+### General guidelines
+
+All pull requests must be fully tested before they're submitted for review.
+
+We use the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) naming scheme for our commit messages, which allows us to generate our [changelog](CHANGELOG.md). Commit messages should include a "type" and a short description (e.g. `fix: add helper methods to show correct trace in sentry (#85)`). For new features, it sometimes makes sense to add a longer description as well. Acceptable commit message types include:
+
+* feat (a new feature)
+* fix (a bug fix)
+* perf (a performance improvement)
+* refactor (a change that improves upon existing code)
+* docs (a change to the docs only)
+* test (a change that only includes tests)
+* chore (a change to CI, etc)
+
+### Fixing bugs
+
+Open a pull request with the patch, and please include a description that clearly describes the problem and solution and references a relevant issue number if applicable. Please also include a test case that covers the previous bug so that the issue won't occur again.
```

Comment:
Probably mention how to submit PR i.e. fork the project and submit PR instead of trying to create a new branch on eris itself. 

1. Create a feature branch off of master so that changes do not get mixed up.
2. Rebase your local changes against the master branch.
3. Run the full project test suite with the go test ./... or makefile test equivalent command and confirm that it passes.


Also, we should the signature part if we are deciding to keep it.

Reply:
good points, i'll work on these sometime tomorrow